machinery-helper/machinery_helper.go

```diff
@@ -356,20 +356,21 @@ func getUnmanagedFilesList(files []string, unmanagedFiles map[string]string, ext
 	unmanagedFilesList := make([]UnmanagedFile, len(unmanagedFiles))
 	i := 0
 	for j := range files {
-		entry := UnmanagedFile{}
-		entry.Name = files[j]
-		entry.Type = unmanagedFiles[files[j]]
+		// only add accessible files
+		if _, err := os.Lstat(files[j]); err == nil || (os.IsNotExist(err) == false && os.IsPermission(err) == false && strings.Contains(err.Error(), "no such device") == false) {
```

Comment:
I think this line is too long. I'd prefer to get rid of the semicolon and do it in two lines.